server/helpers/loadConfig.js

```diff
@@ -0,0 +1,86 @@
+// config intended to be passed on to frontend
+// keys in process .env must be referencesd explicitly
+// so dotenv-webpack is able to include them in the test bundle for Mocha tests
+const check = require('check-types')
+
+// process.env.npm_package_version is undefined on Heroku
+const appVersion = require('../../package.json').version
+
+const {env} = process
+const booleans = ['true', 'false']
+
+check.assert(
```

Comment:
I would rather use check.map

```
check.map({
    foo: 2,
    bar: { baz: 'qux' }
}, {
    foo: check.odd,
    bar: { baz: check.nonEmptyString }
});
```
Then check with `check.all` if everything is true and if not, I would just print the result of check.map from which it should be obvious which value was invalid without the need for such eloquent error messages.